src/oc_erchef/apps/chef_index/src/chef_index.erl

```diff
@@ -0,0 +1,77 @@
+%% Copyright 2015 Chef Software, Inc
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+
+-module(chef_index).
+
+-export([delete/4,
+         add/5,
+         add_async/4,
+         add_async/5
+        ]).
+
+add(TypeName, Id, DbName, IndexEjson, ReqId) ->
```

Comment:
Can we get comments on the difference between `chef_index:add` and `chef_index:add_async`? For one, if we're configured to use RabbitMQ, that's also technically async. What's the difference between the two?